tox.ini

```
@@ -62,11 +62,11 @@ setenv =
     coverage: COVERAGE_FILE={toxworkdir}/coverage.{envname}
     coverage: COVERAGE_PROCESS_START={toxinidir}/.coveragerc
 
-    TRIAL_JOBS={env:TRIAL_JOBS:--jobs=2}
+    TRIAL_JOBS={env:TRIAL_JOBS:}
```

I guess that this is now more of a generic trial argument injector into tox.

```suggestion
    TRIAL_ARGS={env:TRIAL_ARGS:}
```